dashboard/calibration_manager.py

```diff
@@ -38,8 +55,22 @@ def convert_exp_to_sim(self, exp_dict):
             )
             # fill the dictionary
             if exp_name in exp_dict:
-                sim_val = (exp_dict[exp_name] - value["beta"]) * value["alpha"]
-                sim_dict[sim_name] = sim_val
+                if state.use_inferred_calibration:
+                    sim_dict[sim_name] = (
+                        exp_dict[exp_name] - value["beta_guess"]
+                    ) * value["alpha_guess"]
+                    if all(
+                        inferred_key in value.values()
+                        for inferred_key in ["alpha_inferred", "beta_inferred"]
+                    ):
+                        sim_dict[sim_name] = (
+                            exp_dict[exp_name] - value["beta_inferred"]
+                        ) * value["alpha_inferred"]
+                    else:
+                        add_error(
+                            "Inferred calibration does not exist",
+                            "Attempted to use the inferred calibration values to apply to the experimental points but the calibration hasn't been inferret yet. Applying the guess calibration instead.",
+                        )
```

Comment:
I think we usually follow this structure when we add an error, to make sure it shows up both in the GUI and in the terminal logs:
```
title = ...
msg = ...
add_error(title, msg)
print(msg)
```
Could you do the same here?